fatemeh/JS/JS: data fetching + DOM manipulation/index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="indexstyle.css" />
+  </head>
+  <body>
+    <div>
+      <form
+        class="Search"
+        name="Search"
+        method="post"
+        onsubmit="mySearch(event)">
+        <input
+          class="SearchBox"
+          type="text"
+          name="searchTerm"
+          placeholder=" Type here to search..." />
+        <button type="submit" class="SearchButton">Search</button>
+        <button class="cancelButton">Cancel</button>
+      </form>
+      <hr class="line" />
+      <div class="posts"></div>
+    </div>
+
+    <script>
+      let dataArray = [];
+
+      async function fetchData() {
+        document.getElementsByClassName("cancelButton")[0].style.display =
+          "none";
+
+        const response = await fetch(
+          "https://jsonplaceholder.typicode.com/posts"
+        );
+        const data = await response.json();
+        dataArray = data;
+
+        for (let i = 0; i < 10; i++)
+          document.getElementsByClassName(
+            "posts"
+          )[0].innerHTML += `<div class=${i} id=child>
```

Comment:
something I forgot to mention is that instead of selecting the div tag with the class ‘post’ ten times and updating its content, use a variable and when the loop is finished, update the content of ‘post’ once.